examples/np/cem_cartpole.py

```diff
@@ -29,7 +29,7 @@ def cem_cartpole(ctxt=None, seed=1):
     """
     set_seed(seed)
     with LocalTFRunner(snapshot_config=ctxt) as runner:
-        env = GarageEnv(env_name='CartPole-v1')
+        env = GymEnv(env_name='CartPole-v1')
```

Comment:
Since `env` can be passed as a positional argument, it should be called as a positional argument.

Please update all examples like this accordingly.